Dockerfile

```
@@ -0,0 +1,39 @@
+
+# Download the latest installer
+ADD https://astral.sh/uv/install.sh /uv-installer.sh
+
+# Run the installer then remove it
+RUN sh /uv-installer.sh && rm /uv-installer.sh
+
+# Ensure the installed binary is on the `PATH`
+ENV PATH="/root/.local/bin/:$PATH"
+
+# Copy project files
+ADD . /app
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)